youtube_dl/extractor/orf.py

```diff
@@ -280,14 +309,24 @@ class ORFSBGIE(ORFRadioIE):
     api_station = "sbg"
     loop_station = "oe2s"
 
+    _TEST = {
+        'url': 'https://salzburg.orf.at/player/20200423/SGUM',
+        'only_matching': True,
+    }
+
 
 class ORFTIRIE(ORFRadioIE):
-    IE_NAME = 'orf:salzburg'
-    IE_DESC = 'Radio Salzburg'
+    IE_NAME = 'orf:tirol'
+    IE_DESC = 'Radio Tirol'
     _VALID_URL = r'https?://(?P<station>salzburg)\.orf\.at/player/(?P<date>[0-9]+)/(?P<show>\w+)'
```

Comment:
Shouldn't here be Tirol in the URL?

```
_VALID_URL = r'https?://(?P<station>tirol)\.orf\.at/player/(?P<date>[0-9]+)/(?P<show>\w+)'
```

Reply:
True, thanks for catching that. I had tirol duplicated two times and removed the right duplicate unfortunately. Is fixed now.